### PR TITLE
Update wallet footer and add remove token functionality

### PR DESCRIPTION
**Pull request by BMillman19**
## Description

Added the ability to remove tracked tokens from the wallet.

## Motivation and Context

N/A

## How Has This Been Tested?

Manually.

## Types of changes

*   [ ] Bug fix (non-breaking change which fixes an issue)
*   [x] New feature (non-breaking change which adds functionality)
*   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

*   [ ] Change requires a change to the documentation.
*   [ ] Added tests to cover my changes.
*   [ ] Added new entries to the relevant CHANGELOG.jsons.
*   [ ] Labeled this PR with the 'WIP' label if it is a work in progress.
*   [ ] Labeled this PR with the labels corresponding to the changed package.

### CODE REVIEW

File: packages/website/ts/components/portal/portal.tsx [hunk 1]
```diff
@@ -230,14 +244,29 @@ export class Portal extends React.Component<PortalProps, PortalState> {
...
+                    isTracked: false,
+                };
+                this.props.dispatcher.updateTokenByAddress([newToken]);
+            } else {
+                this.props.dispatcher.removeTokenToTokenByAddress(token);
+            }
+            trackedTokenStorage.removeTrackedToken(this.props.userAddress, this.props.networkId, tokenAddress);
+        } else if (isDefaultTrackedToken) {
+            this.props.dispatcher.showFlashMessage(`Cannot remove ${token.name} because it's a default token`);
+        }
```
Comment (fabioberger) [inline packages/website/ts/components/portal/portal.tsx]: Where do we handle `TokenManagementState.Add`?

File: packages/website/ts/components/wallet/wallet.tsx [hunk 2]
```diff
@@ -269,13 +274,31 @@ export class Wallet extends React.Component<WalletProps, WalletState> {
...
+                        </FloatingActionButton>
+                        <div
+                            style={{
+                                paddingLeft: 10,
+                                position: 'relative',
+                                top: '50%',
+                                transform: 'translateY(33%)',
+                            }}
+                        >
+                            add/remove tokens
```
Comment (fabioberger) [inline packages/website/ts/components/wallet/wallet.tsx]: How would `+/- tokens` look? Don't feel strongly, but might be more compact.

File: packages/website/ts/components/portal/portal.tsx (hunk 1 above)
Comment (fabioberger) [reply packages/website/ts/components/portal/portal.tsx]: Ah, I see, maybe just add a TODO comment here?

Comment (BMillman19) [reply packages/website/ts/components/portal/portal.tsx]: seems that the AssetPicker component will automatically track the token

File: packages/website/ts/components/wallet/wallet.tsx (hunk 2 above)
Comment (BMillman19) [reply packages/website/ts/components/wallet/wallet.tsx]: lemme iterate with ben!

**Pull request merged**